webrender/src/device.rs

```diff
@@ -341,6 +341,8 @@ impl TextureId {
             target: gl::TEXTURE_2D,
         }
     }
+
+    pub fn is_invalid(&self) -> bool { *self == TextureId::invalid() }
```

Comment:
Perf-wise, marking it `#[inline]` makes both equally efficient, and `!texture.is_valid()` takes even 1 less char than `texture.is_invalid()`, so it's not like we are sacrificing compactness for clarity here :)